Playbooks/playbook-WildFire_-_Detonate_File_From_URL.yml

```diff
@@ -0,0 +1,364 @@
+    task:
+      id: 5f8eb7ef-2b8d-4c1b-85f9-7622b60f3042
+      version: -1
+      name: Is there a File to detonate?
+      description: Look for file URLs to detonate
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: inputs.FileURL
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 370
+        }
+      }
+    note: false
+  "7":
+    id: "7"
+    taskid: 137e563a-88a3-4ec9-8518-ae4124fca837
+    type: condition
+    task:
+      id: 137e563a-88a3-4ec9-8518-ae4124fca837
+      version: -1
+      name: Is WildFire sandbox enabled?
+      description: Check if integration instance is enabled
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "6"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: WildFire
+                transformers:
+                - operator: getField
+                  args:
+                    field:
+                      value:
+                        simple: state
+            iscontext: true
+          right:
+            value:
+              simple: active
+    view: |-
+      {
+        "position": {
+          "x": 275,
+          "y": 195
+        }
+      }
+    note: false
+  "8":
+    id: "8"
+    taskid: 180e27d9-3b62-44bd-8fd7-0f6ff1e9f346
+    type: regular
+    task:
+      id: 180e27d9-3b62-44bd-8fd7-0f6ff1e9f346
+      version: -1
+      name: WildFire Get Report
+      description: Retrieve results for a file hash using WildFire
+      script: WildFire|||wildfire-report
+      type: regular
+      iscommand: true
+      brand: WildFire
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      format: {}
+      hash:
+        complex:
+          root: WildFire
+          accessor: Report.SHA256
+      md5: {}
+      verbose: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 888
+        }
+      }
+    note: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1085,
+        "width": 605,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: FileURL
+  value:
+    simple: ${FileURL}
```

Comment:
input values should be complex if they are from context

![image](https://user-images.githubusercontent.com/30797606/44846381-dd051f00-ac58-11e8-84d0-4baa434ce783.png)